### PR TITLE
Bugfix: build with other web projects

### PR DESCRIPTION
**Pull request by Zorin95670**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -318,6 +318,32 @@ Running Lidy's tests:
...
+```sh
+# Create a generate-parser.js in [YOUR_PROJECT]/scripts with this content: 
+import('lidy-js/parser/node_parse.js').then(({ preprocess }) => {
+  preprocess('src/lidy/[YOUR_GRAMMAR_NAME].yml');
+});
+
+# Add this line to your package.json script:
+"generate:parser": "node scripts/generate_parser.js"
+
+# Then place your grammar in src/lidy/[YOUR_GRAMMAR_NAME].yml, then run this commands:
```
Comment (k-y0u) [inline README.md]: place the grammar at the good place should be the first step for me.

File: README.md [hunk 2]
```diff
@@ -318,6 +318,32 @@ Running Lidy's tests:
...
+```
+
+Generate parser according to your grammar in your project:
+```sh
+# Create a generate-parser.js in [YOUR_PROJECT]/scripts with this content: 
+import('lidy-js/parser/node_parse.js').then(({ preprocess }) => {
+  preprocess('src/lidy/[YOUR_GRAMMAR_NAME].yml');
+});
+
+# Add this line to your package.json script:
```
Comment (k-y0u) [inline README.md:335]: add it, run it ;)

File: README.md [hunk 3]
```diff
@@ -318,6 +318,32 @@ Running Lidy's tests:
...
+# Then place your grammar in src/lidy/[YOUR_GRAMMAR_NAME].yml, then run this commands:
+npm run generate:parser
+
+# Replace this in the generated js file
+import { parse as parse_input } from '../parser/parse.js'
+# by
+import { parse as parse_input } from 'lidy-js';
+```
+
+
```
Comment (k-y0u) [inline README.md]: 2 lines, too much. (not the same above)

**Review by k-y0u — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Zorin95670) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (Zorin95670) [reply README.md:335]: done

File: README.md (hunk 3 above)
Comment (Zorin95670) [reply README.md]: done

**Review by k-y0u — APPROVED** [review]

**Review by treivax — APPROVED** [review]
How to avoid asking the developer to rewrite the lidy import
(lines 341-344 of the README?).
We need to remove the remaining assertions from the go version of the readme...

**Pull request merged**